libs/cardano-ledger-test/src/Test/Cardano/Ledger/Examples/AlonzoValidTxUTXOW.hs

```diff
@@ -612,7 +620,7 @@ notValidatingWithMintTx pf =
         pf
         [ Inputs' [mkGenesisTxIn 8]
         , Collateral' [mkGenesisTxIn 18]
-        , Outputs' [newTxOut pf [Address (someAddr pf), Amount (MaryValue 0 mint <+> inject (Coin 995))]]
+        , Outputs' [newTxOut pf [Address (someAddr pf), Amount (MaryValue mempty mint <+> inject (Coin 995))]]
```

Comment:
In other files you preferred this format
`Outputs' [newTxOut pf [Address (someAddr pf), Amount (MaryValue (Coin 995) mint)]]`
why leave the inject here>

Reply:
I didn't immediately noticed that `inject` and `<+>` are redundant.

In order to avoid running CI once again, I'll leave it as is. I might fix it some other point.